### ISSUE TITLE
New users are confused by the "Failed to determine the https port for redirect" warning

### ISSUE THREAD

**Issue by danroth27** [dotnet/aspnetcore#53229]
We've received several reports from new users trying out ASP.NET Core that are confused by the "Failed to determine the https port for redirect" warning when running with the default http launch profile:

- https://github.com/dotnet/core/issues/9044
- https://github.com/dotnet/core/issues/9010
- https://github.com/dotnet/core/issues/8960

**Comment by bluesherpa** [comment]
I agree with this confusion. Has it been resolved, or is there some sort of explanation that might help with understanding why the default template is configured this way?
